av-sys/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "av-sys"
```

Comment:
maybe call it `avformat-sys`

Reply:
sure, updated. please help review it again.
thanks